Paintroid/src/main/res/layout/pocketpaint_layout_help_bottom_bar.xml

```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
Please only change the lines you really mean to change.
Why did that many lines just change position without any real changes?

Reply:
I believe those changes were caused by android studios "reformat code on save"- action. I now disabled it so it should not happen in future anymore.